packages/playwright-core/src/server/isomorphic/utilityScriptSerializers.ts

```diff
@@ -186,6 +227,10 @@ export function source() {
       return { u: value.toJSON() };
     if (isRegExp(value))
       return { r: { p: value.source, f: value.flags } };
+    for (const [k, ctor] of Object.entries(typedArrayCtors) as [TypedArrayKind, Function][]) {
```

Comment:
I wonder why do we need `as ...` - doesn't TypeScript infer it based on `typeof typedArrayCtors`?

Reply:
`Object.entries` turns the key into `string`, for whatever reason.